website/docs/r/alert_channel_splunk.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_splunk"
+description: |-
+  Create and manage Splunk Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_splunk
+
+You can use this resource enabled Lacework to forward alerts to Splunk using an HTTP Event Collector.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360007889274-Splunk).
+
+## Example Usage
+
+```hcl
+resource "lacework_alert_channel_" "ops_critical" {
+  name      = "OPS Critical Alerts"
+  hec_token = "AA111111-11AA-1AA1-11AA-11111AA1111A"
+  host = "localhost"
+  port = "80"
+  event_data {
+    index = "index"
+    source = "source"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The Alert Channel integration name.
+* `channel` - (Optional) The Splunk channel name
```

Comment:
could we move this optional argument below to group all required arguments at the top?

Reply:
Also, this bullet is missing a `.` at the end

Reply:
Fixed